source/uwp/SharedVisualizer/ViewModel/HostConfigEditorViewModel.cs

```diff
@@ -17,7 +17,10 @@ namespace AdaptiveCardVisualizer.ViewModel
     public class HostConfigEditorViewModel : GenericDocumentViewModel
     {
         private HostConfigEditorViewModel(MainPageViewModel mainPageViewModel) : base(mainPageViewModel) {
+#if !USE_WINUI3
```

Comment:
wouldn't it be better to check if core windows exist rather than disabling this feature for all WinUI3 use cases?

Reply:
We could do either? I don't think the core window will ever exist in this scenario

Reply:
As a side note, I found a work around for this in the RNW repo, but I'm not sure if its worth implementing for a "nice to have" feature. High contrast selection still works on launch, and the contrast handling is the responsibility of the host app, not the renderer

Reply:
I think using the #if vs null checking CoreWindow comes down to personal preference. The fix on the WinUI side would be removing the underlying reliance on a CoreWindow, rather than putting in a CoreWindow -- which is to say, using the null check wouldn't get us the advantage of suddenly starting to work one day when we update to a newer version of WinUI.

Reply:
Why will a CoreWindow never exist in this previously mentioned scenario?

Reply:
Its unsupported: https://learn.microsoft.com/en-us/windows/apps/desktop/modernize/desktop-to-uwp-supported-api

And the event in question is also listed as unsupported: https://learn.microsoft.com/en-us/windows/apps/desktop/modernize/desktop-to-uwp-supported-api#events